app/views/campaigns/index.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+if @campaign
```

Comment:
This doesn't make sense as an index to me. Its purpose is to return a single campaign, so this would be more like `show`, where we'd have the campaign id in the route. So that would mean calling `/campaigns/1.json` to get campaign 1.

I think that makes more sense than overloading the index method to do very different things in html vs json.